.github/workflows/db-charm-tests.yaml

```diff
@@ -17,13 +17,13 @@ jobs:
       matrix:
         include:
           - charm-repo: canonical/postgresql-operator
-            commit: f298e1ecf07c6e12f4ca6f3afb25ea6b989f0b5a  # 2025-11-22T01:16:13Z
+            commit: 5d96db02fcae63988c7e07982c1512a67e2b6b3d  # 2025-12-05T22:11:12Z
           - charm-repo: canonical/postgresql-k8s-operator
-            commit: f660c4f698f3e0d67b5f3fe9518159f7e7cfa8a2  # 2025-10-23T11:38:44Z
+            commit: 0028c177a941604cbc44dfbfb83a5901b9822673  # 2025-12-02T14:47:20Z
           - charm-repo: canonical/mysql-operator
-            commit: 9dba9415ea55ca2296ee00d13400e8ade45ee3a9  # 2025-11-19T12:26:08Z
+            commit: aecbf0ffb5011df5fd39e3b39383dcaec37f45c3  # rev441 rev440 rev439 2025-12-08T15:00:22Z
```

Comment:
Is there something going on with the multiple "revNNN" here (and below)?

Reply:
[That revision has all of those tags](https://github.com/canonical/mysql-operator/commit/aecbf0ffb5011df5fd39e3b39383dcaec37f45c3).